client/src/components/map.jsx

```diff
@@ -0,0 +1,146 @@
+import styled, { css } from 'styled-components';
+import React, {Component} from 'react';
+
+const MapboxContainer = styled.div`
+  padding: 5px;
+  background: #fff;
+  border: 1px solid #ccc;
+  width: 288.44px;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  margin: 5px;
+`
+
+const Mapbox = styled.img`
+  width: 286px;
+  height: 135px;
+  border: solid 1px;
+`
+
+const AddressBox = styled.div`
+  margin: 5px;
+`
+const AddressList = styled.ul`
+  padding: 5px; 
+  margin: 0px;
+`
+const AddressListItem = styled.li`
+  padding: 0px;
+  list-style-type: none;
+  position: relative
+  margin-bottom: 3px;
+`
+
+const AddressListIcon = styled.span`
+  width: 18px;
+  height: 18px;
+  position: absolute;
+  display: inline-block; 
+  vertical-align: middle; 
+  color: #666;
+`
+const PhoneIcon = styled.span`
+  width: 18px;
+  height: 18px;
+  position: absolute;
+  display: inline-block; 
+  vertical-align: middle; 
+  transform: scaleX(-1);
+  color: #666;
+`
+
+const AddressListText = styled.div`
+  font-size: 14px;
+  margin-left: 30px;
+  display: inline-block;
+  font-weight: bold;
+`
+const PhoneNumberText = styled.div`
+  font-size: 14px;
+  margin-left: 30px;
+  display: inline-block;
+`
+const EditFloat = styled.a`
+  float: right;
+  margin: 1px 3px;
+  font-size: 12px;
+  color: #0073bb;
+  text-decoration: none; 
+  &:hover{
+    text-decoration: underline;
+    cursor: pointer;
+  }
+`
+
+const LinkText = styled.a`
+  margin: 1px 3px;
+  font-size: 14px;
+  color: #0073bb;
+  text-decoration: none; 
+  margin-left: 30px;
+  &:hover{
+    text-decoration: underline;
+    cursor: pointer;
+  }
+`
+
+
+
+const EditIcon = styled.span`
+  margin-right: 4px;
+`
+
+class Map extends React.Component {
+
```

Comment:
since it doesn't have any methods, this should probably just be a functional not a class component